docs/development/contributing_guide.md

```diff
@@ -53,6 +53,13 @@ can find many good git tutorials on the web.
 
```

Comment:
> A recent version of the Rust compiler is needed to build the native modules. The
easiest way of installing the latest version is to use [rustup](https://rustup.rs/).

In the "What do I need?" section above, we could probably move this quote just above the "Synapse can connect to PostgreSQL" paragraph to make it more obvious as a prerequisite as well